src/TrackProgress/SFA.DAS.TrackProgress.Api.AcceptanceTests/testsettings.json

```diff
@@ -3,14 +3,5 @@
   "ConfigNames": "SFA.DAS.TrackProgress.OuterApi",
   "Environment": "LOCAL_ACCEPTANCE_TESTS",
   "Version": "1.0",
-  "APPINSIGHTS_INSTRUMENTATIONKEY": "",
-  "CommitmentsV2InnerApi": {
-    "url": "https://google.com/"
-  },
-  "trackProgressApi": {
-    "url": "https://google.co.uk/"
-  },
-  "coursesApi": {
-    "url": "https://google.co.uk/"
-  }
+  "APPINSIGHTS_INSTRUMENTATIONKEY": ""
```

Comment:
I don't think this testsettings.json file is needed. I removed it and everything ran fine. It seems to duplicate settings elsewhere

Reply:
Removed